### PR TITLE
Explain the implementation of total orders command

### PR DESCRIPTION
**Pull request by GnohChengYi**
Sequence diagrams for total orders command:

TotalOrdersSequenceDiagram1

TotalOrdersSequenceDiagram2

### CODE REVIEW

File: docs/diagrams/TotalOrdersSequenceDiagram2.puml [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+
+MainWindow -> TotalOrdersWindow : reloadData()
+activate TotalOrdersWindow
+
+TotalOrdersWindow -> Logic : getClientTotalOrders()
+activate Logic
+
+Logic --> TotalOrdersWindow
+deactivate Logic
+
```
Comment (ngchisern) [inline docs/diagrams/TotalOrdersSequenceDiagram2.puml:33]: I think there should be a 'deactivate' process here too.
What's wrong: 
The activation bar of the main window still keeps going after the method returns.

**Review by ngchisern — CHANGES_REQUESTED** [review]
Looks good. Just need to fix one mistake.

**Comment by GnohChengYi** [comment]
Updated sequence diagram

TotalOrdersSequenceDiagram2

File: docs/diagrams/TotalOrdersSequenceDiagram2.puml (hunk 1 above)
Comment (GnohChengYi) [reply docs/diagrams/TotalOrdersSequenceDiagram2.puml:33]: Thanks for pointing out!

**Review by ngchisern — APPROVED** [review]
LGTM

**Pull request merged**